examples/docker-compose/docker-compose.yaml

```diff
@@ -140,8 +145,16 @@ services:
     volumes:
       - ./data/mailhog:/maildir
   dashboard:
-    image: nhost/dashboard:0.7.4
-    ports:
-      - "3030:3000"
+    image: nhost/dashboard:0.21.1
+    env_file: .env
```

Comment:
please, provide a working `.env` file and make sure `pnpm e2e`. Thanks!

Reply:
current .env.sample works.